langchain4j-vertex-ai/src/main/java/dev/langchain4j/model/vertexai/VertexAiImageModel.java

```diff
@@ -63,21 +81,82 @@ public enum ImageStyle {
         pop_art
     }
 
+    /**
+     * Supported aspect ratios: 1:1, 9:16, 16:9, 4:3, and 3:4.
+     * The 1:1 square format is the default aspect ratio when not specified.
+     */
+    public enum AspectRatio {
+        SQUARE("1:1"),
+        PORTRAIT("9:16"),
+        LANDSCAPE("16:9"),
+        THREE_FOURTH("3:4"),
+        FOUR_THIRD("4:3");
+
+        private final String ratio;
+
+        AspectRatio(String ratio) {
+            this.ratio = ratio;
+        }
+
+        public String getRatio() {
+            return ratio;
+        }
+    }
+
+    /**
+     * Supported mime types: only PNG and JPEG image formats can be generated.
+     * By default, PNG image files are generated.
+     * When choosing the JPEG image, it is possible to specify a compression level.
+     */
+    public enum MimeType {
+        PNG("image/png"),
+        JPEG("image/jpeg");
+
+        private final String mimeType;
+
+        MimeType(String mimeType) {
+            this.mimeType = mimeType;
+        }
+
+        public String getMimeType() {
+            return mimeType;
+        }
+    }
+
+    /**
+     * Specify whether persons are allowed to be generated.
+     * By default, only adults are allowed.
+     * For all persons, including children, your project will need to be allowlisted.
+     */
+    public enum PersonGeneration {
+        dont_allow,
```

Comment:
I went with lowercase, because they were automatically serialized as strings as is, as the API expects `dont_allow` and not `DONT_ALLOW`. But I guess I just need to define a `toString()` method, and that should do it.

Reply:
Also there was another enum like this with lower-case labels.
It might be a small breaking change though.

Reply:
Lowercase enums is agains Java conventions, I would have a `toString` to serialize it properly as you suggested